context_prop/src/main/java/io/opentelemetry/context/propagation/HttpTextFormat.java

```diff
@@ -29,13 +30,31 @@
  * usually an http request. Propagation is usually implemented via library- specific request
  * interceptors, where the client-side injects values and the server-side extracts them.
  *
+ * <p>Specific concern values (traces, correlations, etc) will be read from the specified {@code
+ * Context}, and resulting values will be stored in a new {@code Context} upon extraction. It is
+ * recommended to use a single {@code Context.Key} to store the entire concern data:
+ *
+ * <pre>{@code
+ * public static final Context.Key CONCERN_KEY = Context.key("my-concern-key");
+ * public MyConcernPropagator implements HttpTextFormat {
+ *   public <C> void inject(Context context, C carrier, Setter<C> setter) {
+ *     Object concernObj = CONCERN_KEY.get(context);
```

Comment:
very tiny nit: call this variable just `concern`

Reply:
Updated ;)